app/src/main/res/layout/movie_details_fragment.xml

```diff
@@ -1,12 +1,205 @@
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_marginTop="@dimen/material_margin_normal"
+                style="@style/ButtonGrey"/>
+
+            <androidx.appcompat.widget.AppCompatRatingBar
+                android:id="@+id/movie_rating"
+                style="@style/Base.Widget.AppCompat.RatingBar.Small"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:isIndicator="true"
+                android:numStars="@integer/rating_num_stars"
+                android:progressBackgroundTint="@color/colorControlNormal"
+                android:stepSize="0.5"
+                app:layout_constraintTop_toTopOf="@id/watch_btn"
+                app:layout_constraintBottom_toBottomOf="@id/watch_btn"
+                app:layout_constraintEnd_toEndOf="parent"
+                tools:rating="5" />
+
+            <TextView
+                android:id="@+id/description_tv"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toBottomOf="@id/watch_btn"
+                style="@style/AppTheme.Text.TextGreyParagraph"
+                android:layout_marginTop="@dimen/material_margin_large"
+                tools:text="In 1985 Maine, lighthouse keeper Thomas Curry rescues Atlanna, the queen of the underwater nation of Atlantis, during a storm. They eventually fall in love and have a son named Arthur, who is born with the power to communicate with marine lifeforms. "/>
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/actors_rv"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toBottomOf="@id/description_tv"
+                android:orientation="horizontal"
+                tools:listitem="@layout/item_actor"
+                app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+                android:layout_marginTop="@dimen/material_margin_normal"
+                tools:itemCount="1"/>
+
+            <TextView
+                android:id="@+id/studio"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="@color/white"
+                android:textStyle="bold"
+                android:textSize="@dimen/text_size_medium"
+                app:layout_constraintTop_toBottomOf="@id/actors_rv"
+                android:layout_marginTop="@dimen/material_margin_normal"
+                app:layout_constraintStart_toStartOf="parent"
+                android:text="@string/movie_details_studio"/>
+
+            <TextView
+                android:id="@+id/genre"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="@color/white"
```

Comment:
Хорошим тоном является именование ресурсов семантически. То есть использовать не сырые цвета (белый/серы итд) а выбирать названия исходя из семантики - то есть где они используются по смыслу.
Например вместо color_white = color_background_main, вместо color_grey = color_divider, color_blue= color_actor_card_background. Это так, FYI. Профит от этого например при добавлении темной темы - названия цветов должны быть универсальны, как я описал выше, а на уровне ресурсов темы они уже меняются. Хорошая статья на эту тему https://habr.com/ru/company/redmadrobot/blog/523916/
Это не обязательно сейчас так делать - но просто опять таки одна из идей для улучшения

Вот даже тут описаны частые ошибки: https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#widgets
Common pitfalls to look out for:

Assuming that the background color is always light
Hardcoding text colors
Setting a hardcoded background color, while using the default text color
Using a drawable icon which is a static color
In all of these cases, use appropriate theme attributes instead of hardcoded colors.